docs/source/name_inference.rst

```diff
@@ -109,6 +109,8 @@ If you don't see an operation listed here, but it would help your use case, plea
    :meth:`Tensor.floor_`,None
    ":meth:`Tensor.frac`, :func:`torch.frac`",:ref:`keeps_input_names-doc`
    :meth:`Tensor.frac_`,None
+   ":meth:`Tensor.gcd`, :func:`torch.gcd`",:ref:`keeps_input_names-doc`
```

Comment:
Let's remove this for now. Named tensor support is on the back burner and these functions don't directly support named tensors since they're implementing a new kernel.